IBPSA/Fluid/Humidifiers/SteamHumidifier_X.mo

```diff
@@ -10,9 +10,7 @@ model SteamHumidifier_X
       final QMax_flow = Modelica.Constants.inf,
       final QMin_flow = -Modelica.Constants.inf,
       final mWatMax_flow = mWatMax_flow,
-      final mWatMin_flow = 0,
-      final energyDynamics = Modelica.Fluid.Types.Dynamics.SteadyState,
```

Comment:
Assuming this removal is intentional.

Reply:
Yes, this is intentional as `energyDynamics` is assigned in the base class now (https://github.com/ibpsa/modelica-ibpsa/pull/1591/files#diff-70ded3e6ed7a3a11eee46a4517919a9762e70e4855632fde06dd590b2258ddcfR35). Previously, it was missing in the base class, and extending models corrected this by assigning it one level higher.

Reply:
This change is reasonable. Thanks for all changes.